src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -1674,18 +1678,24 @@ mod tests {
         {
             let mut f = OpenOptions::new().write(true).open(devnode).unwrap();
             // Write 1 more sector than is initially allocated to a pool
-            let write_size = datablocks_to_sectors(INITIAL_DATA_SIZE) + Sectors(1);
+            let write_size = datablocks_to_sectors(DEFAULT_INITIAL_DATA_SIZE) + Sectors(1);
```

Comment:
With the changes in this PR, this line, too, should use the actual data size rather than than the ```DEFAULT_INITIAL_DATA_SIZE``` value in order for the test to achieve its purpose in the same way as befofre.